src/blueprint/from_config.rs

```diff
@@ -168,13 +170,19 @@ fn to_interface_type_definition(definition: ObjectTypeDefinition) -> Valid<Defin
 }
 fn to_fields(type_of: &config::Type, config: &Config) -> Valid<Vec<blueprint::FieldDefinition>> {
   let fields: Vec<Option<blueprint::FieldDefinition>> = type_of.fields.iter().validate_all(|(name, field)| {
+    let field_type = &field.type_of;
+
+    if !is_scalar(field_type) && config.find_type(field_type).is_none() && config.find_union(field_type).is_none() {
```

Comment:
```suggestion
    if !is_scalar(field_type) && config.contains(field_type) {
```
Can add a helper method `contains`.

Reply:
done